components/builder-api/src/depot/server.rs

```diff
@@ -2654,6 +2827,94 @@ fn do_cache_response(response: &mut Response) {
     )));
 }
 
+fn set_memcache_package(
```

Comment:
It would be great if you could pull the memcache functions to a new file instead of leaving them in server.rs.  Also create a 'struct MemCache' and put the functions into the struct.  This will make it much easier to integrate into the new framework.  Then you could just do something like:

```
MemCache::new(config).set_package(&ident);
```

Reply:
Also, it would be great if you could hook up a couple of metrics in the MemCache struct so we can monitor the caching get/sets on our dashboard.